spec/steps/show_me_the_mails_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "steps/show_me_the_mails"
+
+describe Spreewald::Steps::ShowMeTheMails do
```

Comment:
Are there tests for the `MailFinder` or a feature with more than one mail?

You could check if the output is formatted correctly then.

Reply:
Added a spec :)